vue.config.js

```diff
@@ -25,7 +34,6 @@ module.exports = {
     devtool: 'source-map',
     resolve: {
       alias: {
-        vue: path.resolve('./node_modules/vue'),
```

Comment:
We don't always want to resolve to this node_modules, we might want to resolve at the parent's Vue instance.